prow/plugins/override/override.go

```diff
@@ -304,6 +304,28 @@ type descriptionAndState struct {
 	state       string
 }
 
+// Parses /override foo something
+func parseOverrideInput(in string) []string {
+	quoted := false
+	f := strings.FieldsFunc(in, func(r rune) bool {
+		if r == '"' {
+			quoted = !quoted
+		}
+		return !quoted && r == ' '
+	})
+	retval := []string{}
+	for _, val := range f {
+		if val[0:1] == `"` {
+			retval = append(retval, val[1:len(val)-1])
+			fmt.Println(val[1 : len(val)-1])
+		} else {
+			retval = append(retval, strings.TrimRight(val, "\r"))
+		}
```

Comment:
```suggestion
		retval = append(retval, strings.Trim(strings.TrimSpace(val), `"`))
```
I think this should do